packs/vsphere/actions/vm_hw_nic_edit.yaml

```diff
@@ -0,0 +1,24 @@
+---
+  name: "vm_hw_nic_edit"
```

Comment:
yes.. the aim of the action was to be a starting point. eventually it will be able to do more. for now i just need a network change action. No reason it can't be expanded to allow the update of other configuration information about the nic.